src/assets/persons.json

```diff
@@ -6074,7 +6074,22 @@
 		  "state": "",
 		  "country": "RU"
   }
-}
-
+},
+  {
+    "id": 380,
+    "name": "Lucas Sallada",
+    "img": "https://avatars3.githubusercontent.com/u/18669601?s=460&v=4",
+    "links": {
+      "website": "http://sallada.org/",
+      "linkedin": "https://www.linkedin.com/in/lucas-sallada-6242a4106/",
+      "github": "https://github.com/yonlu"
+    },
+    "jobTitle": "Web Developer",
+    "location": {
+      "city": "Beaver Falls",
+      "state": "PA",
+      "country": "USA"
+    }
+  },
```

Comment:
There appears to be an additional comma at the end of the object containing your data. A trailing comma would cause a syntax error while parsing JSON files, because unfortunately JSON doesn't allow trailing commas.